packages/flutter_test/lib/src/finders.dart

```diff
@@ -548,26 +572,66 @@ abstract class MatchFinder extends Finder {
 }
 
 class _TextFinder extends MatchFinder {
-  _TextFinder(this.text, { bool skipOffstage = true }) : super(skipOffstage: skipOffstage);
+  _TextFinder(
+    this.text, {
+    this.skipRichText = true,
+    bool skipOffstage = true,
+  }) : super(skipOffstage: skipOffstage);
 
   final String text;
 
+  /// Whether standalone [RichText] widgets should be skipped or not.
+  ///
+  /// Defaults to `true`.
+  ///
+  /// If enabled, only [Text] widgets will be considered. These of course also
+  /// insert a [RichText] widget, however, that is ignored and so are all
+  /// standalone [RichText] widgets.
```

Comment:
This sentence is hard to understand. Maybe rephrase?

Reply:
Sure, I can see it :)